### PR TITLE
Implement client booking commands

### PR DESCRIPTION
**Pull request by LimZiJia**

### CODE REVIEW

**Comment by LimZiJia** [comment]
Fixes #90

File: src/main/java/seedu/address/logic/commands/BookingCommand.java [hunk 1]
```diff
@@ -81,77 +89,138 @@ public BookingCommand(String actionWord, Index index, int bookingToDeleteIndex)
      * @param actionWord "list"
      * @param index of housekeeper whose bookings to list
      */
-    public BookingCommand(String actionWord, Index index) {
+    public BookingCommand(String type, String actionWord, Index index) {
```
Comment (GERARDJM018) [inline src/main/java/seedu/address/logic/commands/BookingCommand.java:92]: Nice implementation! now can use BookingCommand for both type

File: src/main/java/seedu/address/logic/commands/BookingCommand.java [hunk 2]
```diff
@@ -81,77 +89,138 @@ public BookingCommand(String actionWord, Index index, int bookingToDeleteIndex)
...
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        switch (actionWord) {
-        case "add":
-            requireNonNull(model);
-            List<Housekeeper> lastShownListAdd = model.getFilteredHousekeeperList();
-
-            if (housekeeperIndex.getZeroBased() >= lastShownListAdd.size()) {
-                throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        requireNonNull(model);
```
Comment (GERARDJM018) [inline src/main/java/seedu/address/logic/commands/BookingCommand.java:109]: Nice defensive programming!

**Review by GERARDJM018 — APPROVED** [review]
Nice!

**Pull request merged**